pkg/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+	"github.com/containers/image/signature"
+	"github.com/sirupsen/logrus"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() {
+	registriesConfPath := sysregistries.RegistriesConfPath(nil)
+	_, err := os.Stat(registriesConfPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Errorf("Required file registries.conf not found in %s, %s", registriesConfPath, err)
+		} else {
+			logrus.Errorf("error checking registries.conf in %s, %s", registriesConfPath, err)
+		}
+	}
+
+	mountsDefaultFilePath := "/usr/share/containers/mounts.conf"
+	mountsOptionalFilePath := "/etc/containers/mounts.conf"
+	_, errDefault := os.Stat(mountsDefaultFilePath)
+	_, errOptional := os.Stat(mountsOptionalFilePath)
+	if errDefault != nil && errOptional != nil {
+		if os.IsNotExist(errDefault) && os.IsNotExist(errOptional) {
+			logrus.Errorf("Required file mounts.conf not found in %s or %s.", mountsDefaultFilePath, mountsOptionalFilePath)
+		} else {
+			logrus.Errorf("error checking mounts.conf in %s and %s, %s, %s", mountsDefaultFilePath, mountsOptionalFilePath, errDefault, errOptional)
+		}
+	}
+
+	seccompDefaultPath := "/usr/share/containers/seccomp.json"
+	seccompOverridePath := "/etc/crio/seccomp.json"
+	_, errDefault = os.Stat(seccompDefaultPath)
+	_, errOptional = os.Stat(seccompOverridePath)
+	if errDefault != nil && errOptional != nil {
+		if os.IsNotExist(errDefault) && os.IsNotExist(errOptional) {
+			logrus.Errorf("Required file seccomp.json not found in %s or %s, please ensure that the containers-common package has been installed.", seccompDefaultPath, seccompOverridePath)
```

Comment:
We need a path, not just the file.  BTW The user can override this file, so we need to check the default path or the override path.  Not sure if this is required at all, since I think the tool should fail if this file does not exist.

Reply:
not all commands fail. `buildah run` fails, but `buildah from` or `buildah pull` can work
the override path can be checked inside the function for the command. 
only need to check the files required for all command?

Reply:
Lets talk about buildah run failing when their is no seccomp file.  I think we could just warn.